### PR TITLE
Add support for LB with DNS hostname only

### PR DESCRIPTION
**Pull request by fl42**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] I have read the **CONTRIBUTING** document.
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

AWS Load Balancer (NLB) now uses DNS-based hostname. This PR adds support to extract hostname from Kubernetes Service.

### CODE REVIEW

**Comment by utkuozdemir** [comment]
Thanks for the PR. Looks good to me. Can you please fix the linter errors? You can use `task fmt` and `task lint` for that (you'll need [this](https://taskfile.dev/)).

**Comment by fl42** [comment]
Thanks, fixed :+1:

File: internal/k8s/service.go [hunk 1]
```diff
@@ -59,7 +59,12 @@ func GetServiceAddress(
 			}
 
 			if len(res.Status.LoadBalancer.Ingress) > 0 {
-				result = res.Status.LoadBalancer.Ingress[0].IP
+
```
Comment (utkuozdemir) [inline internal/k8s/service.go:62]: Can you remove this empty line? Linter is still complaining about it:
> Error: unnecessary leading newline (whitespace)

**Comment by fl42** [comment]
Sorry, I did the quick fix using online IDE ;)

**Comment by utkuozdemir** [comment]
Thanks for the contribution :)

**Pull request merged**